cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -240,6 +240,8 @@ public class KafkaCluster extends AbstractModel implements SupportsMetrics, Supp
         }
     }
 
+    private KafkaMetadataStateManager.KafkaMetadataConfigurationState kafkaMetadataConfigState;
+
```

Comment:
I guess this should be with the other private fields of similar use for example after the line 214?